ssm/agents.py

```diff
@@ -328,7 +329,13 @@ def run_receiver(protocol, brokers, project, token, cp, log, dn_file):
                     if protocol == Ssm2.STOMP_MESSAGING:
                         ssm.send_ping()
 
-            except (NotConnectedException, AmsConnectionException) as error:
+            except (
```

Comment:
The issue with this indentation is that lines 333 to 336 are the same level as 339+.

Could we just go for this style, on two lines, please? (Similar to what you did on line 40.
```
# Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                         var_three, var_four)
```

Reply:
Done with addressing the issue. And, Applied the rebase.